src/summary/documentation/dimensions-card.component.tsx

```diff
@@ -6,17 +6,26 @@ import SummaryCardRow from "../cards/summary-card-row.component";
 import SummaryCardRowContent from "../cards/summary-card-row-content.component";
 import ShowMoreCard from "./show-more-card.component";
 import styles from "./dimensions-card-level-one.css";
+import { useCurrentPatient } from "@openmrs/esm-api";
+
 export default function DimensionsCard(props: DimensionsCardProps) {
   const [dimensions, setDimensions] = React.useState([]);
   const [showMore, setShowMore] = React.useState(false);
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
 
   React.useEffect(() => {
-    const sub = getDimensions(props.currentPatient.id).subscribe(dimensions => {
+    const sub = getDimensions(patientUuid).subscribe(dimensions => {
       setDimensions(dimensions);
     });
 
     return () => sub.unsubscribe();
-  }, []);
+  }, [patientUuid]);
```

Comment:
This was a missing dependency in the React effect's array. It was causing the dimensions to be refetched every time that this component re-rendered.